src/aggregations/pipeline-aggregations/moving-function-aggregation.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+const isNil = require('lodash.isnil');
+
+const PipelineAggregationBase = require('./pipeline-aggregation-base');
+
+const ES_REF_URL =
+    'https://www.elastic.co/guide/en/elasticsearch/reference/master/search-aggregations-pipeline-movfn-aggregation.html';
+
+/**
+ * Given an ordered series of data, the Moving Function aggregation
+ * will slide a window across the data and allow the user to specify
+ * a custom script that is executed on each window of data.
+ * For convenience, a number of common functions are predefined such as min/max, moving averages, etc.
+ *
+ * `moving_fn` aggregations must be embedded inside of a histogram or
+ * date_histogram aggregation.
+ *
+ * [Elasticsearch reference](https://www.elastic.co/guide/en/elasticsearch/reference/master/search-aggregations-pipeline-movfn-aggregation.html)
```

Comment:
```suggestion
 * [Elasticsearch reference](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-pipeline-movfn-aggregation.html)
```

Reply:
Which version of Elasticsearch was this aggregation added in? Could we mention that as a `NOTE` in the documentation section here? As far as I know, it is only available in Elasticsearch 6.4.0+.

Reply:
Yes, it seems like it: https://github.com/elastic/elasticsearch-net/pull/3419

Reply:
I added a NOTE about the version :)